pkg/apply/applier.go

```diff
@@ -245,7 +215,7 @@ func (a *Applier) Run(ctx context.Context, invInfo inventory.InventoryInfo, obje
 	return eventChannel
 }
 
-type Options struct {
+type ApplierOptions struct {
```

Comment:
We have `DestroyerOptions`, so renamed this for consistency as suggested here https://github.com/kubernetes-sigs/cli-utils/pull/485#discussion_r755460760.